tests/unit/amo/components/TestEditableCollectionAddon.js

```diff
@@ -76,4 +83,164 @@ describe(__filename, () => {
     sinon.assert.called(clickEvent.stopPropagation);
     sinon.assert.calledWith(removeAddon, addon.id);
   });
+
+  describe('notes area', () => {
+    it('hides the notes area by default', () => {
+      const root = render();
+
+      expect(root.find('.EditableCollectionAddon-notes')).toHaveLength(0);
+    });
+
+    it('shows the read-only version of the notes area if there are notes', () => {
+      const notes = 'Some notes.';
+      const addon = {
+        ...fakeAddon,
+        notes,
+      };
+      const root = render({ addon });
+      const notesArea = root.find('.EditableCollectionAddon-notes');
```

Comment:
Should we add an assertion for this `notesArea`? (like `toHaveLength(1)`?)

Reply:
Well, we use it a number of times in the test after that to find more things, so that seems like maybe overkill. The test will fail if it's not found, but I guess we'll get a better error if we include that assertion. I'll add it.